src/components/directory/StateLinks.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState, useEffect } from "react";
+import { DirectoryProfile } from "../../types/entities";
+import DirectoryList from "./DirectoryList";
+
+export type RegionEntitiesResponse = {
+  count: number;
+  docs: DirectoryProfile<never>[];
+};
+
+interface StateLinksProps {
+  baseUrl: string;
+};
+
+const StateLinks = (props: StateLinksProps) => {
+  const { baseUrl } = props;
+  const title = "Find a TurboTax Verified Pro in All States";
+
+  const [regionEntities, setRegionEntities] = useState<RegionEntitiesResponse>();  
+  const streamsApiKey = "5606fbbd0d232b9a2a1a81a6e489255b";
```

Comment:
Is there a way we can add this to the .env file or something instead of hard coding it here ?

Reply:
Yes, we can add this Streams API key to both site settings and .env file.

Reply:
https://github.com/ttverifiedpro/itvp-pro-website/pull/8